enterprise/server/remote_execution/execution_server/execution_server_test.go

```diff
@@ -0,0 +1,157 @@
+package execution_server_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/go-redis/redis/v8"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"gorm.io/gorm"
+
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/remote_execution/execution_server"
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/scheduling/task_router"
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/testutil/testredis"
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/util/redisutil"
+	"github.com/buildbuddy-io/buildbuddy/server/interfaces"
+	"github.com/buildbuddy-io/buildbuddy/server/tables"
+	"github.com/buildbuddy-io/buildbuddy/server/testutil/testenv"
+
+	"github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
+)
+
+type schedulerServerMock struct {
+	interfaces.SchedulerService
+}
+
+func (s *schedulerServerMock) CancelTask(ctx context.Context, taskID string) (bool, error) {
+	return true, nil
+}
+
+func executionServer(t *testing.T, env *testenv.TestEnv) *execution_server.ExecutionServer {
+	redisTarget := testredis.Start(t).Target
+	rdb := redis.NewClient(redisutil.TargetToOptions(redisTarget))
+	env.SetRemoteExecutionRedisClient(rdb)
+	env.SetRemoteExecutionRedisPubSubClient(rdb)
+
+	router, err := task_router.New(env)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
fyi, you can also use `require.NoError(t, err)` here -- a bit more concise but no need to change it if you prefer this style, just throwing it out there as an option

Reply:
Done